compiler/src/test/scala/org/scalajs/nscplugin/test/BinaryCompatTest.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.nscplugin.test
+
+import org.scalajs.nscplugin.test.util._
+import org.junit.Test
+
+// scalastyle:off line.size.limit
+
+class BinaryCompatTest extends DirectTest with TestHelpers  {
```

Comment:
Have you considered and AST compiler test checking presence/absence of relevant default accessors as an alternative?

Reply:
Indeed, that's much better.